Newsletters: add 214 (2022-08-24)

harding opened the pull request:
- [x] Releases/RCs & topic entries @harding 
- [x] Client/Services section @bitschmidty 
- [x] Bitcoin Core 25504 @Xekyo

glozow commented on _posts/en/newsletters/2022-08-24-newsletter.md:
Unless I'm missing something?
```suggestion
  and switches to using [taproot][topic taproot] addresses everywhere
```

harding replied:
This is a quote from their release notes where elided some details and inserted the conjunction.  I put the modification in brackets to indicate it's not original (i.e., this is deliberate and not releated to markdown).

glozow commented on _posts/en/newsletters/2022-08-24-newsletter.md:
I have a hard time parsing "using something applicable by default"

```suggestion
  by default".
```

bitschmidty commented on _posts/en/newsletters/2022-08-24-newsletter.md:
perhaps ```suggestion
  ,`parent_descs`, the associated descriptors. Additionally,
```

bitschmidty commented on _posts/en/newsletters/2022-08-24-newsletter.md:
perhaps 
```suggestion
  as defined by [BOLTs #759][].  The PR indicates that this change is in
```

bitschmidty commented on _posts/en/newsletters/2022-08-24-newsletter.md:
added or removed "from" it?

bitschmidty commented on _topics/en/silent-payments.md:
```diff
@@ -0,0 +1,56 @@
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Silent payments** are a type of payments that can be made to a
```
```suggestion
  **Silent payments** are a type of payment that can be made to a
```

harding commented:
Made suggested edits (thanks @glozow and @bitschmidty !), review other contributions (thanks @Xekyo and @bitschmidty !), updated releases/RCs, and added topic links.  Thanks everyone!

bitschmidty commented:
Looks good, thank you @harding @Xekyo and @glozow for your contributions this week!
